### ISSUE TITLE
[BUG] win_wua.list functions fail on win server installs regression with 3002

### ISSUE THREAD

**Issue by TheBigBear** [saltstack/salt#58837]
**Description**
All of the win_wua.list functions fail on most windows servers minions. If I downgrade salt to pervious version 3001 then the win_wua cmds work again. 

**Setup**
upgrade of salt windows server minion to 3002 breaks win_wua.list

**Steps to Reproduce the behavior**
```
salt OVH-AADCONNECT win_wua.list install=True
```
results in
```
OVH-AADCONNECT:
    'win_wua' __virtual__ returned False: WUA: The Background Intelligent Transfer service (bits) must not be disabled
```

**Expected behavior**
It should have listed and installed available windows updates

**Screenshots**

**Versions Report**
<details><summary>salt --versions-report</summary>
(Provided by running salt --versions-report. Please also mention any differences in master/minion versions.) 

```
OVH-AADCONNECT:
    Salt Version:
               Salt: 3002
     
    Dependency Versions:
               cffi: 1.12.2
           cherrypy: 17.4.1
           dateutil: 2.8.0
          docker-py: Not Installed
              gitdb: 2.0.5
          gitpython: Not Installed
             Jinja2: 2.10.1
            libgit2: Not Installed
           M2Crypto: Not Installed
               Mako: 1.0.7
       msgpack-pure: Not Installed
     msgpack-python: 1.0.0
       mysql-python: Not Installed
          pycparser: 2.19
           pycrypto: Not Installed
       pycryptodome: 3.9.8
             pygit2: Not Installed
             Python: 3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 20:34:20) [MSC v.1916 64 bit (AMD64)]
       python-gnupg: 0.4.4
             PyYAML: 5.3.1
              PyZMQ: 18.0.1
              smmap: 2.0.5
            timelib: 0.2.4
            Tornado: 4.5.3
                ZMQ: 4.3.1
     
    System Versions:
               dist:   
             locale: cp1252
            machine: AMD64
            release: 2016Server
             system: Windows
            version: 2016Server 10.0.14393 SP0
```
</details>

**Additional context**
If I downgrade back to 3001 then it all works as expected again. 
The slightly odd thing is that out of a mixed bunch 30 servers ( some core installs and some full GUI as old as 2012R2 all the way up to 2016 and 2019 it works even with salt minion 3002 on 4 out of 30 servers). 

I could not find a correlation of certain server releases or core or GUI installs that all failed or all succeeded, can't tell what makes 4 out of my 30 act differently and have a still working win_wua.list even on salt minion version 3002. 

But it fails with same error message on 26 out of 30 using slat minion 3002 and works on 30 out of 30 using slat minion 3001.  Let me know if you needs debug output or anything else I can provide to help find the problem. @twangboy is this to do with the win_wua fixes you added recently to deal with win 10 pro 2004 and 2009 windows updates issues 13f57328e2880afdc9804d6fff1c522e5d5c296d?

**Comment by cmcmarrow** [comment]
@TheBigBear I don't think @twangboy commit broke `__virtual__`

**Comment by cmcmarrow** [comment]
but this one done by @twangboy sure did https://github.com/saltstack/salt/commit/0487e49b4d2d4c426b527e20b9a08863b141ff35

**Comment by cmcmarrow** [comment]
Thanks @TheBigBear. I got the same error in my box as well.

**Comment by FlosCarmeli** [comment]
I too am having the same problem after I upgraded to 3002. I'll have to downgrade to resolve this issue since I use Salt for all my Windows updates.

**Comment by twangboy** [comment]
This should be working in Salt 3002.2

**Comment by TheBigBear** [comment]
I see it as fixed in 3002.5, as well. It was definitely still broken in 3002 and unfortunately, we somehow missed 3002.2 altogether Thanks @twangboy.
